akka-http/src/main/scala/akka/http/scaladsl/server/ContentNegotation.scala

```diff
@@ -22,9 +22,9 @@ final class MediaTypeNegotiator(requestHeaders: Seq[HttpHeader]) {
       Accept(mediaRanges) ← requestHeaders
       range ← mediaRanges
     } yield range).sortBy { // `sortBy` is stable, i.e. upholds the original order on identical keys
```

Comment:
Could you please remove the comment (which is a remnant of the previous implementation logic) and update the ScalaDoc?